packages/features/portfolio/src/containers/portfolio-overview/containers/leverage-vaults-overview.tsx

```diff
@@ -69,6 +69,15 @@ const LeverageVaultsOverview = ({ data }: IProps) => {
   );
 };
 
+const Header = styled(H2)(({ theme }) => ({
```

Comment:
I will ask Isaac to match this to the style library rather than doing something custom.

Reply:
@anazari96 switch all the headings into H4